packages/ui/src/views/connectors/connectors-list/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { LogoName } from '@components/logo'
+
+import { ConnectorConfigType } from '../types'
+
+export const ConnectorTypeToLogoNameMap: Map<ConnectorConfigType, LogoName> = new Map([
```

Comment:
`Record<ConnectorConfigType, LogoName>` might be a cleaner way to manage this

Reply:
Will cleanup in a follow up PR.